### PR TITLE
[Docs] Add header for files

### PR DESCRIPTION
**Pull request by BIGWangYuDong**
Follow https://github.com/open-mmlab/mmcv/pull/1181
- add `# Copyright (c) OpenMMLab. All rights reserved.` at the head of the files(exclude config files)
- add `add_copyright_header` file in tools/misc

### CODE REVIEW

**Comment by jshilong** [comment]
Please resolve conflicts first

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2018-2019 Open-MMLab. All rights reserved.
```
Comment (ZwwWayne) [inline LICENSE:1]: time can be changed from `2018-2019` to `2018-2023`

**Review by ZwwWayne — APPROVED** [review]

**Review by jshilong — APPROVED** [review]
LGTM

**Comment by ZwwWayne** [comment]
we do not need to keep the script in tools/ because we only need add_copyright.py once.

**Comment by BIGWangYuDong** [comment]
Done

**Pull request merged**